pkg/runtime/iota.go

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 The NanoBus Authors.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package runtime
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+	"github.com/nanobus/nanobus/pkg/config"
+
+	"github.com/nanobus/nanobus/pkg/oci"
+)
+
+type iotaKey struct{}
+
+func FromContext(ctx context.Context) IotaConfig {
```

Comment:
How will this be used? I was thinking something along these lines for how actions under an iota can resolve files, but wasn't thinking the entire config struct.